setup.cfg

```diff
@@ -1,5 +1,2 @@
-[bdist_wheel]
-universal = 1
```

Comment:
Sure, universal wheels are those which:

* contain no C extensions
* work on both Python 2 and 3

https://wheel.readthedocs.io/en/stable/user_guide.html

But because only Python 3 is supported, we no longer need this config.

Reply:
Thanks